lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -41,9 +41,10 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
       description: 'Properly size images',
       informative: true,
       helpText:
-        'Image sizes served should be based on the device display size to save network bytes. ' +
-        'Learn more about [responsive images](https://developers.google.com/web/fundamentals/design-and-ui/media/images) ' +
-        'and [client hints](https://developers.google.com/web/updates/2015/09/automating-resource-selection-with-client-hints).',
+        'Serve images that are smaller than the user\'s viewport to save cellular data ' +
+        'and improve load time. Consider using responsive images or client hints for the ' +
```

Comment:
I actually think we should just stick to @patrickhulce's idea to restrict the `helpText` to just the "why" of the audit, and leave the "how" in the reference doc (when it's available)

When a reference doesn't yet exist, then we can link to "how" pages, such as client hints

Otherwise we're going to endlessly go back and forth on when we should include "how" content, and when we should omit